client/src/components/rooms/RoomsMeetingExperience.test.tsx

```diff
@@ -61,9 +80,12 @@ describe('RoomsMeetingExperience', () => {
     expect(callComposite.first().props().callInvitationUrl).toBe('testUrl');
   });
 
-  it('sets display name as Virtual appointments Virtual appointments User and should have not have invite url if the participant is a attendee', async () => {
+  it('sets display name as Virtual appointments Virtual appointments User and should not have invite url if the participant is a attendee', async () => {
```

Comment:
nit: "Virtual appointments" is typed twice

Reply:
:) Thanks.. Fixed this!